brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -278,6 +279,9 @@ public void build(ExtFile appDir, File outFile)
         mAndRes.setVersionInfo(meta.versionInfo);
         mAndRes.setSharedLibrary(meta.sharedLibrary);
 
+        if (meta.sdkInfo != null && meta.sdkInfo.get("minSdkVersion") != null)
```

Comment:
Can you use braces for added logic? I try to stay away from single statement control flow as its just prone to errors.